libertyequalitydata/oura-motivational-quotes
# MyQoutes Widget

Hi there 👋🏻

Thanks for taking the time to review the project. Here are some things essential to note about it:

## Now into the project...
 > This is my first task as a frontend developer intern at Prifina Company

### Project Description:
A widget that displays on its interface, motivational quotes based on the last 4 days of  ActiveSummary data as well as showing an average of the score, rest time, time without using the ring, and total calories.

### Widget screenshot
![MyQoutes](https://user-images.githubusercontent.com/73354132/192031811-48b46e7e-1b4e-4439-bdc7-4b3c736e8431.PNG)

### Steps for Testing the Widget

1. You can either fork or clone `https://github.com/Nicolaswg/oura-motivational-quotes.git` in this repo.
2. The package manager by default is `Yarn`. To set up the project go to the root path of the repo and run `yarn install`.
3. It will be good to run the same command of step 2 inside this path `./packages/myQoutes`.
4. Finally you can run `yarn start` in the same path as step 3 for launching your localhost.
5. In case you one to check Storybook the available command is `yarn storybook`


## Final notes 📔 
It is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

<img src="https://media0.giphy.com/media/6brH8dM3zeMyA/giphy.gif"/>
